core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -179,6 +181,7 @@ private[deploy] class Master(
     }
     persistenceEngine = persistenceEngine_
     leaderElectionAgent = leaderElectionAgent_
+    startupTime = System.currentTimeMillis()
```

Comment:
I think @guoxiaolongzte refers to how long has it been since last restart of master by "startupTime".............

Reply:
well, my question is what's the difference for a master which just started 10 mins ago and 1 hour ago in terms of stability?

Reply:
Spark master process zombie, the background has a shell script automatically pull the spark master process to ensure high availability, but the restart process, there may be some applications such as failure.

If I look at startup time metric today, if the startup time is ten days ago or a month ago, I would think the system is relatively stable, there is no restart behavior.

If I look at the startup time metric today, if startup time was 1 day ago or an hour ago, I would assume that the system is unstable and that a recent reboot has occurred, requiring developers to troubleshoot problems and analyze them.

Reply:
> Spark master process zombie, the background has a shell script automatically pull the spark master process to ensure high availability, but the restart process, there may be some applications such as failure.

I didn't understand what you are going to express, but just a guess, you have some external script to monitor the availability of master process, if it disappears, the script would restart it?

if this is the case, why not output last restart time in your own monitoring system since you can get this easily in that "script" instead of leaking to Spark's code base and additionally you always need to query Spark UI for the timestamp?

Reply:
I understand what you mean.
I do not always query, but developers occasionally to observe.

Reply:
I would vote -1 for this as not a solid use case...but it's my personal opinion, 


times for others